### PR TITLE
9th homework is done

### PR DESCRIPTION
**Pull request by MayHoax**

### CODE REVIEW

File: R_D_HW_9/homework_9_generator.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    while True:
+        yield a
+        a, b = b, a + b
+
+while True:
+    fib = fibonacci_generator()
+    n = int(input("Enter a number: "))
+    for i in range(n):
+        next_fib = next(fib)
+    print(next_fib)
```
Comment (klymenok) [inline R_D_HW_9/homework_9_generator.py]: Your program print a wrong number in the sequence. For example, for 7 it should print 13 but it prints 8.

**Review by klymenok — CHANGES_REQUESTED** [review]

**Comment by MayHoax** [comment]
My bad, the sequence had an error with initial values. I used a sequence that had two 1s at the beginning.

**Comment by MayHoax** [comment]
Now it's should return 13 if 7 given

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**